### DISCUSSION TITLE
Conversion of EBSD file from .osc to .ctf

### DISCUSSION THREAD

**Discussion by sjain-ml** [mtex-toolbox/mtex#1909] [Ask Anything]
I got EBSD results in .osc format. Now I'm not able to open it in MTEX. How I ca =n use .osc files in MTEX?

**Comment by kilir** [comment]
Hi,
there are different version of osc-file formats, so I assume yours is not supported by loadEBSD_osc being either too old or too new. Save your data as .ang on your vendor software is probably the easiest solution.
Cheers,
Rüdiger
